sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -106,6 +106,9 @@ function getContentType(body: any): string | undefined {
 
 export interface InternalRequestParameters extends RequestParameters {
   responseAsStream?: boolean;
+
+  /** Skip serialization option for string body request */
```

Comment:
Maybe we can add a bit of context on why this is here. Like "this option allows customers to handle serialization, for example stringifying JSON on their own without the core library trying to serialize again"